spring-cloud-dataflow-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -36,7 +36,7 @@ One can enable, disable these features by setting the following boolean properti
 * `spring.cloud.dataflow.features.tasks-enabled`
 * `spring.cloud.dataflow.features.analytics-enabled`
 
-By default, all these features are enabled for `local` dataflow server.
+By default, all these features except 'analytics' features are enabled.
```

Comment:
don't need "features" 2x in this sentence (probably best to remove the 2nd occurrence)